spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -1583,6 +1583,79 @@
       end
     end
 
+    describe "line 53c checkbox" do
+      context "when taxpayer indicated all members of household have health insurance" do
+        let(:submission) {
+          create :efile_submission, tax_return: nil, data_source: create(
+            :state_file_nj_intake,
+            eligibility_all_members_health_insurance: "yes",
+            )
+        }
```

Comment:
[dust] - we could also do `allow_any_instance_of(Efile::Nj::Nj1040Calculator).to receive(:calculate_line_53c_checkbox).and_return true` etc in the PDF tests, since going from the intake to calculator is already tested in the calculator tests.